### PR TITLE
Improve styles with less version

### PR DESCRIPTION
**Pull request by GrimLink**

### CODE REVIEW

File: view/frontend/web/css/source/module/_dots.less [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    button {
+        display: block;
+        width: @notorama-dot-size;
+        height: @notorama-dot-size;
+        padding: 0;
+        border-radius: 50%;
+        border: @notorama-dot-border;
+        background-color: @notorama-dot-color;
+        box-shadow: none;
+        text-indent: -100vw;
```
[the remark this answers is not in the document]
Comment (GrimLink) [reply view/frontend/web/css/source/module/_dots.less]: Ah thx 

> This is causing artifacts on mobile:
> 
> image
> 
> Would it be better to use `font-size: 0;`? Not sure if that affects accessibility.

Both options are bad for accessibility.
Stupid of me, i could have easily added an span with and screan-reader only, styles

**Comment by GrimLink** [comment]
> Thanks for this, I appreciate porting these to both LESS and SASS. I've noticed that the loader overlay is gone when the page initially loads. I whitewashed the gallery out slightly while the JavaScript was loading to match the rest of Magento. Similar to the 'Add to Cart' button below.
> 
> **Before JS initialised**
> image
> **After JS initialised**
> image
> 
> Was there a reason you removed that? I don't like having spinners/loaders everywhere, but I think it would be beneficial to have one here to make it obvious the a user that the gallery is not yet interactable. Maybe positioning a loader over the carousel, as not to block the main image would help.

I did not intend to remove it.
I forgot an `position: relative;` which broke it 😅

**Comment by GrimLink** [comment]
@robaimes No problem 😉
Thank you for creating this module.

The bugs you mentioned are fixed now.
Stupid little things I missed when splitting your code in chunks 😅

**Comment by robaimes** [comment]
> @robaimes No problem 😉
> Thank you for creating this module.
> 
> The bugs you mentioned are fixed now.
> Stupid little things I missed when splitting your code in chunks 😅

No worries 😄 i'll take a look tomorrow

File: view/frontend/styles/module/_video.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+    }
+}
+
+.iframe-wrapper {
+    position: relative;
+
```
[the remark this answers is not in the document]
Comment (GrimLink) [reply view/frontend/styles/module/_video.scss]: Good one will check and correct this

File: view/frontend/styles/module/_video.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        right: 0;
+        bottom: 0;
+        left: 0;
+        width: $notorama-video-btn-size;
+        height: $notorama-video-btn-size;
+        margin: auto;
+        padding: 1rem;
+        border-radius: 50%;
+        background:
+        rgba(black, .8)
```
[the remark this answers is not in the document]
Comment (GrimLink) [reply view/frontend/styles/module/_video.scss]: Ah must have missed that.
Normally Stylelint and prettier handle this for me, spoiled by tools 😅

**Comment by robaimes** [comment]
@GrimLink Thanks, merged! :+1:

**Pull request merged**